netty/src/test/java/io/grpc/netty/NettyHandlerTestBase.java

```diff
@@ -221,6 +222,7 @@ protected final Http2Connection connection() {
     return handler().connection();
   }
 
+  @CanIgnoreReturnValue
```

Comment:
Same reason as in `WriteQueue`, not all callers consume the future.  Netty's API usually returns a future that is passed in.  It looks like this code is trying to be like that.